From ea27b932b58cb6cf9d2f1ad1eddd187ee34b9a90 Mon Sep 17 00:00:00 2001
From: Sylvestre Ledru <sylvestre@debian.org>
Date: Fri, 13 Sep 2019 09:31:19 +0000
Subject: [PATCH] Fix a perl warning: Scalar value @ArgParts[0] better written
 as $ArgParts[0] at /usr/share/clang/scan-build-10/libexec/ccc-analyzer line
 502.

llvm-svn: 371832
---
 clang/tools/scan-build/libexec/ccc-analyzer | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clang/tools/scan-build/libexec/ccc-analyzer b/clang/tools/scan-build/libexec/ccc-analyzer
index 6d24a1af4539..800f38b5ba24 100755
--- a/clang/tools/scan-build/libexec/ccc-analyzer
+++ b/clang/tools/scan-build/libexec/ccc-analyzer
@@ -499,7 +499,7 @@ my $HasSDK = 0;
 foreach (my $i = 0; $i < scalar(@ARGV); ++$i) {
   my $Arg = $ARGV[$i];
   my @ArgParts = split /=/,$Arg,2;
-  my $ArgKey = @ArgParts[0];
+  my $ArgKey = $ArgParts[0];
 
   # Be friendly to "" in the argument list.
   if (!defined($ArgKey)) {
